sdk/test-utils/recorder-new/src/index.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export { TestProxyHttpClient } from "./core-v2-recorder";
-export { TestProxyHttpClientCoreV1 } from "./core-v1-recorder";
+export { recorderHttpPolicy, RecorderClient } from "./core-v2-recorder";
+export { TestProxyHttpClientCoreV1, RecorderClientCoreV1 } from "./core-v1-recorder";
```

Comment:
Do we still need to export `TestProxyHttpClientCoreV1`? It seems that outside world only needs a `HttpClient` now.

Reply:
removed